ci-operator/step-registry/ovn/sdn-migration/feature-migration/validate-egress-ip/ovn-sdn-migration-feature-migration-validate-egress-ip-commands.sh

```diff
@@ -23,25 +28,26 @@ validate_sdn_egressip_crs () {
   kubectl get hostsubnet $HOSTSUBNET_NAME -o json | jq .egressIPs | tee current_hsn_egressIPs
   kubectl get netnamespace $NETNAMESPACE_NAME -o json | jq .egressIPs | tee current_nns_egressIPs
 
-  echo "$EXPECT_HOSTSUBNET_EGRESS_CIDRS" | tee expected_egressCIDRs
-  echo "$EXPECT_HOSTSUBNET_EGRESS_IPS" | tee expected_hsn_egressIPs
-  echo "$EXPECT_NETNAMESPACE_EGRESS_IPS" | tee expected_nns_egressIPs
+  expected_egressCIDRs=$(oc get no  "$HOSTSUBNET_NAME" -o jsonpath="{.metadata.annotations.cloud\.network\.openshift\.io/egress-ipconfig}" | jq -r '.[].ifaddr.ipv4')
+  ip_part=$(echo "$expected_egressCIDRs" | cut -d'/' -f1)
+  expected_egressIPs="[\"${ip_part%.*}.5\"]"
+  expected_egressCIDRs="[\"$expected_egressCIDRs\"]"
 
-  if diff <(jq -S . current_egressCIDRs) <(jq -S . expected_egressCIDRs); then
+  if diff <(jq -S . current_egressCIDRs) <(jq -S <<< "$expected_egressCIDRs"); then
```

Comment:
All rollback/reverse migration tests failed here. The variable in `jq -S <<< "$expected_egressCIDRs"` cannot be parsed correctly.

Reply:
Fixed, jq is funny sometiems